cql/ManageSpecialPopulation.cql

```diff
@@ -236,16 +246,18 @@ define ShouldSwitchToRiskAt25Text:
 define RecommendationForPatientsYoungerThan25:
   case
     // Heading 3: Management of Histology of Less than CIN2 Preceded by Cytology ASC-H and HSIL in Patients Younger than 25 Year
-    when ( // Heading 3.3
+    // ===========    
+    when ( // K.1 Heading 3 case 1
```

Comment:
Same sentiment as comment below but for this rec - original cytology would likely not be the 2nd most recent, for reason explained blow.

Reply:
concur - switched to use PrecedingCytologyIsHsil and PrecedingCytologyIsAscH instead.